core/src/components/textarea/test/label-placement/textarea.e2e.ts

```diff
@@ -302,4 +302,23 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, co
       await expect(page.locator('.container')).toHaveScreenshot(screenshot(`textarea-multi-line-sizing`));
     });
   });
+  test.describe(title('textarea: floating/stacked label layering'), () => {
+    test('label should not be covered by text field', async ({ page }) => {
```

Comment:
Let's add a `testInfo.annotations` to track the reason behind this test.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27870/commits/9ffc1d933c2d295ed832c23162f0b741679e903e